### PR TITLE
parameters_for supports enum

### PR DESCRIPTION
**Pull request by osiro**

### CODE REVIEW

File: spec/support/controller_parameter_support.rb [hunk 1]
```diff
@@ -3,9 +3,26 @@ module ControllerParameterSupport
   # FactoryGirl.attributes_for ignores associations, which is problematic for us. So we make
   # our own implementation.
   def parameters_for(*factory_args)
-    all_attributes = FactoryGirl.build(*factory_args).attributes.with_indifferent_access
+    all_attributes = sanitize_attributes(FactoryGirl.build(*factory_args))
     all_attributes = all_attributes.reject {|attr_name| attr_name.in?(["id", "created_at", "updated_at", "deleted_at"]) }
```
Comment (jordanmaguire) [inline spec/support/controller_parameter_support.rb]: I feel like this line should be part of the sanitize attributes workflow

File: spec/support/controller_parameter_support.rb [hunk 2]
```diff
@@ -3,9 +3,26 @@ module ControllerParameterSupport
...
+
+    # Replaces enums integers with strings.
+    # E.G:
+    # instance = User.last
+    # instance.attributes => { id: 1, status: 0 }
+    # sanitize_attributes(instance) => { id: 1, status: 'active' }
+    #
+    instance.defined_enums.each do |enum, values|
+      attributes[enum] = values.key(attributes[enum])
+    end
```
Comment (jordanmaguire) [inline spec/support/controller_parameter_support.rb]: I would put this in its own function, personally.

```ruby
attributes = sanitize_enums(attributes)
```

or something

**Comment by jordanmaguire** [comment]
:+1:  I :heart: it great job

**Comment by swebb** [comment]
:+1:

File: spec/support/controller_parameter_support.rb (hunk 2 above)
Comment (osiro) [inline spec/support/controller_parameter_support.rb]: Done!

**Pull request merged**